TheOracle2/GameObjects/Move.cs

```diff
@@ -0,0 +1,59 @@
+namespace TheOracle2.GameObjects;
```

Comment:
again, probably needs to be removed. I'm not sure if this is a rabbit hole we want to start down, but I don't think it is.

Reply:
> again, probably needs to be removed. I'm not sure if this is a rabbit hole we want to start down, but I don't think it is.

gah, i thought i deleted this! reason #534513 not to do such huge PRs, rsek. i made it when i was avoiding working with the database. then i realized that working with the db wasn't nearly so difficult as i assumed.